packages/app/src/app/pages/NewDashboard/Components/SandboxCard/index.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { sandboxUrl } from '@codesandbox/common/lib/utils/url-generator';
+import { Element, Link } from '@codesandbox/components';
+
+export const SandboxCard = ({ sandbox }) => (
+  <Element>
+    <Link href={sandboxUrl({ id: sandbox.id, alias: sandbox.alias })}>
+      {sandbox.title || sandbox.alias || sandbox.id}
```

Comment:
You can use `getSandboxName` from `common/utils` instead
```suggestion
      {getSandboxName(sandbox)}
```